website/static/js/tests/comment.test.js

```diff
@@ -0,0 +1,133 @@
+/*global describe, it, expect, example, before, after, beforeEach, afterEach, mocha, sinon*/
```

Comment:
It tells [JSHint](http://jshint.com/docs/#inline-configuration) that those variables are global and defined elsewhere.